docs/Bioinformatics-Skills/Kids-First/Upload_Data.md

```diff
@@ -0,0 +1,382 @@
+
+You can either create a new project or choose an existing project.
+
+#### New project
+Create a new Cavatica project by clicking on the "Projects" tab on the Cavatica homepage and selecting the " + Create a project" option. You can name your new project whatever you like.  Remember your project name and replace "project-name" with the name of your project in the code block below.
+
+
+#### Existing project
+Alternatively, you may choose to select an existing project. To get a list of all existing projects to choose from, run this code:
+
+=== "AWS Instance Code"
+  ```
+  uploader -t a??????????????????????????????? --list-projects
+  ```
+
+The `-t` flag tells AWS to look for an Authentication token. Remember to replace `a???????????????????????????????` with your own Authentication token.
+
+!!! Important
+    If you have `_`s in your project name, replace them with `-` in the uploader code.
+
+### Step 5c: Moving Files
+
+Finally, you can transfer files by running the following code. Remember to replace "project-name" with the name of your project and "username" with your cavatica login name.
+
+=== "AWS Instance Code"
+    ```
+    cd ~/fastq
+    uploader -t a??????????????????????????????? -p username/project-name *.fastq.gz
+    ```
+=== "Expected Output"
+
+    ```
+    Initializing upload...
+    Starting upload of 2 item(s) to 'project-name'
+    Uploading file '/home/ubuntu/fastq/ERR458493.fastq.gz'
+    5fadb1cee4b05495de67d7ea	/home/ubuntu/fastq/ERR458493.fastq.gz    100%
+    Uploading file '/home/ubuntu/fastq/ERR458494.fastq.gz'
+    5fadb1dce4b05495de67d7f1	/home/ubuntu/fastq/ERR458494.fastq.gz   45.52%
+    ```
+
+By running this code, you are moving files from your AWS instance to the "project-name" project within Cavatica. The `*` wildcard copies all the files with extension ".fastq.gz". The `-p` flag tells AWS which Cavatica project to put the files into. If you wish, you could use the `-f` flag to create a subfolder inside the project (not shown here).
+
+You're all done! Log in to Cavatica and look for your files in the the "Files" tab in your project.
+
+
+## Step 6: Edit Metadata
+
+!!! Important
+    The name of the example project we are using in this section of the tutorial is called "sim_fastq". Your project name will be different. To practice, we recommend following along using the practice yeast fastq files ("ERR458493.fastq.gz" and "ERR458494.fastq.gz") that you previously pushed to the "project-name" project.
+
+- First, click into your desired Project and then select the "Files" tab.
+
+![](images/Files_Tab.png "Files Tab")
+
+- You should see all your files listed here.
+
+- Next, select the files whose metadata you wish to edit by checking the box next to the file name.
+
+![](images/Select_Samples.png "Select the Files")
+
+- Click the "Edit Metadata" button
+
+![](images/Edit_Metadata_Button.png "Edit Metadata Button")
+
+- You should see a pop-up window on the right side of the screen
```

Comment:
It seems that Cavatica has set metadata fields that you can edit. Is there a way to add new metadata terms if not listed in the options ?

Reply:
I'm 98% sure that the answer is no, but I will double check.

Reply:
nevermind there is a roundabout way of doing this. significant addition coming up.

Reply:
@abhijna i can test this tutorial when you're done adding this metadata part

Reply:
postponed to the next release. it's not working right now. will fuss with it some at a later time.